src/browser-installer/constants.ts

```diff
@@ -20,6 +20,7 @@ export const MIN_CHROMIUM_MAC_ARM_VERSION = 93;
 export const MIN_CHROMIUM_VERSION = 73;
 export const MIN_FIREFOX_VERSION = 60;
 export const MIN_EDGEDRIVER_VERSION = 94;
+export const DRIVER_WAIT_INTERVAL = 25; // 20ms
```

Comment:
is the comment correct? I don't understand how 25 == 20ms?

Reply:
> is the comment correct?
No